### ISSUE TITLE
No Packages from `environment.yml` are installed?

### ISSUE THREAD

**Issue by michaelweinold** [brightway-lca/brightway-hub#18]
```
  __  _____ _   _ ___
  \ \/ / _ \ | | / __|
   >  <  __/ |_| \__ \
  /_/\_\___|\__,_|___/
​
  xeus-python: a Jupyter kernel for Python
  Python 3.10.2

import peewee

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/xpython_42/3073019139.py in <cell line: 1>()
----> 1 import peewee

ModuleNotFoundError: No module named 'peewee'
```

 - [x] https://github.com/jupyterlite/xeus-python-demo/issues/28

**Comment by michaelweinold** [comment]
...resolved with https://github.com/brightway-lca/brightway-hub/issues/17. Maybe if _some_ packages are not found (`bw2io=0.9.dev21` from `conda-forge`), _none_ are installed? Also, JupyterLite seems to work best in Chrome for now.

**Comment by michaelweinold** [comment]
The issue seems to persist:

```

  __  _____ _   _ ___
  \ \/ / _ \ | | / __|
   >  <  __/ |_| \__ \
  /_/\_\___|\__,_|___/
​
  xeus-python: a Jupyter kernel for Python
  Python 3.10.2
import peewee
import peewee
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import peewee

ModuleNotFoundError: No module named 'peewee'
```

**Comment by michaelweinold** [comment]
This is a major blocker (with little hope of resolution without help from the Executablebooks/QuantStack team). Some ideas to escalate the issue:

 - [ ] QuantStack Matrix Chat: [My Question](https://matrix.to/#/!bxEXnseMFYzltyRExC:gitter.im/$mu68-0x0G7ANk_ERTOrwERllb7RS9Zr9_WRNAduOcxk?via=gitter.im&via=matrix.org&via=salt-rock-lamp.ems.host)
 - [ ]

**Comment by michaelweinold** [comment]
...evidently resolved in [`empack=3.1.2`](https://github.com/emscripten-forge/empack/releases/tag/3.1.2). `conda-forge` feedstock updated currently pending action of the [regro-cf-autotick-bot](https://github.com/conda-forge/empack-feedstock/commits?author=regro-cf-autotick-bot).
